DXMainClient/DXGUI/Generic/TopBar.cs

```diff
@@ -161,7 +161,7 @@ public override void Initialize()
             lblTime = new XNALabel(WindowManager);
             lblTime.Name = "lblTime";
             lblTime.FontIndex = 1;
-            lblTime.Text = "99:99:99";
+            lblTime.Text = Renderer.GetSafeString(new DateTime(1, 1, 1, 23, 59, 59).ToLongTimeString(), lblDate.FontIndex);
```

Comment:
`lblDate.FontIndex`, shouldn't it rather be `lblTime.FontIndex`?

Reply:
Oh, thanks. I just directly copied the ```Renderer.GetSafeString(DateTime.Now.ToShortDateString(), lblDate.FontIndex);``` line here.

Reply:
Fixed in 2dc65ac55dc69804628fc78f29b2f03dceadf1b5 .
Please squash the two commits when merging this PR, thank you.